sdk/core/core-tracing/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Breaking Changes
 
 - Removed `OpenCensusSpanWrapper` and `OpenCensusTracerWrapper` from the public API. Customers using these wrappers should migrate to using `OpenTelemetry` directly. [PR #15770](https://github.com/Azure/azure-sdk-for-js/pull/15770)
+- Update `@azure/core-tracing` to version 1.0.0-preview.12. This brings core-tracing up to date with `@opentelemetry/api@1.0.0`.
+  - Breaking changes are internal, and supported by `@azure/core-http@1.2.7`
```

Comment:
What are the implications of this line? Is it worth mentioning (only because if it's completely invisible to the user it might be better to omit it).

Reply:
It's the Span#context change to Span#spanContext -> which actually would be useful information if you are using the Span directly. I can add that